src/views/NutritionView.vue

```diff
@@ -131,9 +135,9 @@ const getProducts = async (nbOfPages) => {
   const randomPage = Math.floor(Math.random() * nbOfPages);
   const {
     data: { products },
-  } = await axios(
-    `${OFF_URL}/state/photos-validated/state/nutrition-facts-to-be-completed/${randomPage}.json?fields=code,lang,image_nutrition_url,product_name`
-  );
+      } = await axios(
+      `${OFF_URL}/state/photos-validated/state/nutrition-facts-to-be-completed/${randomPage}.json?fields=code,lang,image_nutrition_url,product_name`
+    );
```

Comment:
If you do not ask for the `images` field here, you will never be able to get the image id of the nutrition picture and so you can not build the URL of the OCR

See https://github.com/openfoodfacts/openfoodfacts-hungergames/issues/359#issuecomment-1072300252

Reply:
> If you do not ask for the `images` field here, you will never be able to get the image id of the nutrition picture and so you can not build the URL of the OCR
> 
> See [#359 (comment)](https://github.com/openfoodfacts/openfoodfacts-hungergames/issues/359#issuecomment-1072300252)

I have tried creating the URL in the same format. This is the result:
https://world.openfoodfacts.org/api/v0/product/4640028621441.json?fields=product_name,images

I can't find the `nutrition_pt` in the result to further read `imgid`.

Reply:
> I can't find the nutrition_pt in the result to further read imgid.

In this case this si because your product only have english images. So you have nutrition_en.
Last two letters are the language code.

https://pt.openfoodfacts.org/api/v2/product/5602691253281.json?fields=product_name,images this one has a protugese image for nutrition.

Reply:
> In this case this si because your product only have english images. So you have nutrition_en.

Thanks a lot. I didn't know suffix after `_` stood for a language. 

Is there a way to find what image belongs to what language? Because the next image appears at random.